Integrations/integration-ArcSightESM.yml

```diff
@@ -325,7 +325,7 @@ script:
                 Body: body
             },
             params.insecure,
-            params.useproxy
+            params.proxy
```

Comment:
Was it never worked?

Reply:
It seems not. But because we don't have proxy env so it never tested